src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -152,6 +152,10 @@
           short-summary: Service principal used for authentication to Azure APIs.
           long-summary:  If not specified, a new service principal is created and cached at
                          {sp_cache} to be used by subsequent `az aks` commands.
+        - name: --skip-subnet-role-assignment
+          type: bool
+          short-summary: Skip role assignment for subnet (advanced networking).
+          long-summary:  If it is true, please make sure your service principal has the access to your subnet.
```

Comment:
If specifying this, please make...

Reply:
Oops sorry I meant the long-summary text itself. Full sentence:
`If specifying this, please make sure your service principal has the access to your subnet.`

Reply:
ok, got you